cai/tools/web/bug_bounty_platforms.py

```
@@ -0,0 +1,376 @@
+                "weakness_id": vulnerability_type
+            }
+        }
+    }
+    
+    try:
+        if attachments:
+            # Prepare multipart form data: JSON 'data' field and file uploads
+            multipart_data = {"data": json.dumps(data)}
+            files_payload = []
+            for path in attachments:
+                files_payload.append(("attachments[]", open(path, "rb")))
```

Ruff (0.11.9)

Use a context manager for opening files

(SIM115)